apps/web/.env.production

```diff
@@ -70,3 +70,11 @@ LOG_LEVEL_FILE=debug
 # The logging level for the stdout.
 # Options: trace, debug, info, warn, error, fatal, silent
 LOG_LEVEL_STDOUT=info
+
+## apps/api ###################################################################
+
+# Required values as api appeal formatters imported by the web tests are are
+# coupled to config files (for some reason).
+# TODO: remove the requirement for these props in the web .env config
+DATABASE_URL="sqlserver://"
```

Comment:
You shouldn't need the DATABASE_URL

Reply:
so this is a carry over from our env.test, @lutamKC added it there, I'm assuming we might nee them here as well?
@lutamKC can you confirm

Reply:
removed the vars for now